CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.4.3 (unreleased)
+
+- Gutenberg has changed name to REPLACE_ME!
```

Comment:
ah, interesting!

Reply:
See https://github.com/Keats/gutenberg/issues/377 for the discussion.
The new name will be `zola`

Reply:
hehe, interesting... we got a famous celebrity (musician, actor, presenter) in South Africa using that as a nick